blog/.vuepress/config.js

```diff
@@ -13,8 +13,8 @@ module.exports = {
         text: "Events",
         items: [
           {
-            text: "Next Event",
-            link: "/events/index.html"
+            text: "Upcoming Event",
+            link: "" // Defined at compile time in blog/.vuepress/plugins/sidebar/create.js
```

Comment:
Currently computed to the first next events, it would be nice to have a page `/events/upcoming.html` that list all upcoming events.